app/javascript/react/components/MovieShow.js

```diff
@@ -2,8 +2,8 @@ import React from "react"
 
 const MovieShow = (props) => {
   return(
-    <div>
-      <h2>{props.title}</h2>
+    <div class="callout secondary cell small-6">
+      <h2 class="field">{props.title}</h2>
```

Comment:
We will need to use `className` instead of `class` while in React since `class` is a restricted keyword